ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/apk/ApkCreator.kt

```diff
@@ -94,6 +98,27 @@ class ApkCreator(private val rootDir: File) {
         return SdkLocator.getSdkDirectory(rootDir, issueReporter).toPath()
     }
 
+    /**
+     * Grab the debug keystore when creating the split apks. Doing this step makes sure
+     * the corresponding /META-INF/BNDLTOOL.SF and *.RSA files are created in the apks.
+     **/
+    private fun getAndroidDebugKey(): SigningConfiguration {
+        val androidDebugKey = File(System.getProperty("user.home"), ".android/debug.keystore")
+        val keystorePassword = "android"
+        val keyAlias = "androiddebugkey"
+
+        if (!androidDebugKey.exists()) {
+            throw java.lang.RuntimeException("Android debug keystore not found at ${androidDebugKey.path}")
+        }
```

Comment:
Not sure if we should fail or if we should just log and continue if we can't find the key. I guess the debug key should always be available in all cases? If that's true then failing could be ok. If there are valid cases where the key is missing then maybe logging and continuing is the better approach.

Looks like some tests are failing because of this, so either it's a test setup issue or it's valid.